warehouse/models/mart/gtfs_quality/_mart_gtfs_quality.yml

```diff
@@ -207,6 +212,9 @@ models:
       - *check
       - name: status
       - name: feature
+      - name: matches_entity
```

Comment:
If this is being surfaced on this mart table can you add a short description indicating that this is just for testing/auditing? Same for below